tests/unit/services/insights-test.js

```diff
@@ -0,0 +1,165 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Service | insights', function (hooks) {
+  setupTest(hooks);
+
+  hooks.beforeEach(function () {
+    this.insightsService = this.owner.lookup('service:insights');
+  });
+
+  test('default interval settings', function (assert) {
+    let intervalSettings = this.insightsService.getIntervalSettings({ week: {subInterval: '1min'} });
+    let intervalSettings2 = this.insightsService.getIntervalSettings();
+    intervalSettings2.week.subInterval = '1min';
+
+    assert.ok(this.insightsService);
+    assert.deepEqual(intervalSettings, intervalSettings2);
+  });
+
+  test('dates from interval', function (assert) {
+    let serviceDates = this.insightsService.getDatesFromInterval('week');
+
+    assert.equal(Array.isArray(serviceDates), true);
+    assert.equal(serviceDates.length, 2);
+
+    let [start, end] = serviceDates;
+    assert.equal(start < end, true);
+
+    let day = 1000 * 60 * 60 * 24;
+    let week = day * 7;
+    let serviceDatesDifference = (end - start);
+    let diff = week - serviceDatesDifference;
+    assert.ok(diff <= day && diff >= (day * -1));
+  });
+
+  test('active repos with no metric data', async function (assert) {
+    this.server.create('user');
+
+    let result = await this.insightsService.getActiveRepos({id: 1, '@type': 'user'}, 'week');
```

Comment:
Wouldn't the following work here?
```javascript
const user = this.server.create('user');
const result = await this.insightsService.getActiveRepos(user, 'week');
```

Reply:
Nice, yeah it certainly appears to. Change made. Thanks!